File: app/src/main/java/agi/erecreditsmanager/LectureWorld/LectureWorldLayout.java
package agi.erecreditsmanager.LectureWorld;

import android.content.Context;
import android.view.LayoutInflater;
import android.widget.RelativeLayout;
import android.widget.TextView;

import agi.erecreditsmanager.R;

public class LectureWorldLayout extends RelativeLayout {

    Context context;
    TextView lectureWorldCreditsTextView;

    public LectureWorldLayout(Context context) {
        super(context);
        this.context = context;

        init();
    }

    private void init() {
        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        inflater.inflate(R.layout.lectureworld, this, true);

        lectureWorldCreditsTextView = (TextView) findViewById(R.id.lectureWorldCreditsTextView);
    }

    public void setLectureWorldCreditsTextView(String text) {
        lectureWorldCreditsTextView.setText(text);
    }
}

File: app/src/main/java/agi/erecreditsmanager/LectureWorld/LectureWorldLayout.kt
package agi.erecreditsmanager.LectureWorld

import agi.erecreditsmanager.R
import android.content.Context
import android.view.LayoutInflater
import android.widget.RelativeLayout
import kotlinx.android.synthetic.main.lectureworld.view.*

class LectureWorldLayout(context : Context) : RelativeLayout(context) {
    init {
        LayoutInflater.from(context).run {
            inflate(R.layout.lectureworld, this@LectureWorldLayout, true)

            lectureWorldCreditsTextView
        }
    }

    fun setLectureWorldCreditsTextView(text : String) {
        lectureWorldCreditsTextView.text = text
    }
}